content/release-10-15-0/streaming-analytics-10-15-0-bundle/10_15_0_4.md

```diff
@@ -0,0 +1,33 @@
+---
+weight: 36
+title: Release 10.15.0.4
+layout: redirect
+---
+
+This release upgrades the Apama-ctrl microservice to use Apama 10.15.0.3.
```

Comment:
Apama version is 10.15.0.2

Reply:
Thanks, changed it to "In this release, the Apama-ctrl microservice uses the same Apama version as in the previous 10.15.0.3 release."